src/Schema/DefinitionBuilder.php

```diff
@@ -246,9 +246,12 @@ protected function buildObjectType(ObjectTypeDefinitionNode $node): ObjectType
         return newObjectType([
             'name'        => $node->getNameValue(),
             'description' => $node->getDescriptionValue(),
-            'fields'      => function () use ($node) {
+            'fields'      => $node->hasFields() ? function () use ($node) {
```

Comment:
I noticed an inconsistency in how fields were handled compared to e.g. interfaces, so I decided to change this.